database/seeds/UpdatePreloadedColumnToH5PLibrariesSeeder.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+
+class UpdatePreloadedColumnToH5PLibrariesSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        DB::table('h5p_libraries')
+            ->where('name', 'H5P.CurrikiInteractiveVideo')
+            ->update([
+            			'preloaded_js' => 'dist/h5p-interactive-video.js',
```

Comment:
Correct formatting in database/seeds/UpdatePreloadedColumnToH5PLibrariesSeeder.php